src/clr-icons/shapes/all-shapes.ts

```diff
@@ -24,20 +44,9 @@ const allShapesSets = [
   TravelShapes,
   ChartShapes,
   TextEditShapes,
-];
-
-const allShapes: any = {};
-
-for (const set of allShapesSets) {
-  for (const shape in set) {
-    if (set.hasOwnProperty(shape)) {
-      allShapes[shape] = set[shape];
-    }
-  }
-}
-
-if (typeof window !== 'undefined' && window.hasOwnProperty('ClarityIcons')) {
-  window.ClarityIcons.add(allShapes);
-}
+]
+  .map(set => Object.keys(set).map(shape => ({ shape, value: set[shape] })))
+  .reduce((shapes, shape) => [...shapes, ...shape], [])
+  .reduce((shapes, { shape, value }) => ({ ...shapes, [shape]: value }), {} as AllShapesType);
```

Comment:
Isn't it overcomplicating things here? As far as I can see, this can be simply: 
```
const allShapes: any = {
  ...CoreShapes,
  ...CommerceShapes,
  ...EssentialShapes,
  ...MediaShapes,
  ...SocialShapes,
  ...TechnologyShapes,
  ...TravelShapes,
  ...ChartShapes,
  ...TextEditShapes
};

```

Reply:
You're right, I tried to kind of match the old logic with better techniques instead of just optimizing it 100%. Fixed in the latest push, thanks for the suggestion!